yaml/base/calico-vpp.yaml

```diff
@@ -74,6 +74,7 @@ data:
       full-coredump
       cli-listen /var/run/vpp/cli.sock
       pidfile /run/vpp/vpp.pid
+      exec /etc/vpp/startup.exec
```

Comment:
Might be added for tests no ?

Reply:
I added it there because it makes one less thing to configure when we need it (for instance for some debugging), and it doesn't hurt to have it if it's empty.

Reply:
Right, works :)